extensions/wikia/Wall/WallController.class.php

```diff
@@ -49,7 +49,7 @@ public function messageDeleted() {
 	 *
 	 * @desc Renders old User_talk:[username] page in new place, using Wall_renderOldUserTalkPage.php template
 	 *
-	 * @author Andrzej 'nAndy' √Ö¬Åukaszewski
+	 * @author Andrzej 'nAndy' Łukaszewski
```

Comment:
Why confused? I think it is actually pretty funny to see a P2 fix that corrects a misspelled name :-)